tests/quic/testIntegrationUsecase.nim

```diff
@@ -72,28 +75,25 @@ suite "Quic integration usecases":
       checkEqual(message, receivedData)
 
       await stream.close()
-      await connection.close()
       serverWg.done()
 
-    proc runClient() {.async.} =
-      let client = makeClient()
-      let connection = await client.dial(address)
+    asyncSpawn accept(listener, handleServerConn)
 
+    proc runClient(connection: Connection) {.async.} =
```

Comment:
nit: consider `runClient` -> `handleClientConn` similarly to `handleServerConn` in same test